test/rules.jl

```diff
@@ -1,6 +1,7 @@
 #######
 # Demo setup
 using StaticArrays: @SVector
+import ChainRulesCore: frule
```

Comment:
BlueStyle says it is prefer to do overloading via explit namespace, rather than via `import`
`function ChainRulesCore.frule(...)`

Reply:
Huh. Good to know. Will fix globally.